graphql/resolvers.go

```diff
@@ -2527,6 +2527,14 @@ func (r *annotationResolver) UserCanModify(ctx context.Context, obj *restModel.A
 
 }
 
+func (r *annotationResolver) WebhookConfigured(ctx context.Context, obj *restModel.APITaskAnnotation) (bool, error) {
+	t, err := r.sc.FindTaskById(*obj.TaskId)
+	if err != nil {
+		return false, err
```

Comment:
This should be an `InternalServerError`.
Theres also a potential for `t` to be nil so we should handle that as well